matrix/main.py

```diff
@@ -34,18 +35,41 @@ def configLogging(options):
 
 
 def setup(matrix, args=None):
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(
+        description="Run a local bundle through a suite of tests to ensure "
+                    "that it can handle the types of operations and failures "
+                    "that are common for all deployments.",
+        epilog="A default suite of tests will always be run, unless -D "
+               "is given.  Additional suites can be passed in as "
+               "arguments.  Additionally, the bundle will be checked for "
+               "a matrix.yaml file which will be included as an "
+               "additional suite if present.  All suites will be merged "
+               "together, and suites containing tests with the same name as a "
+               "test in a previous suite will override that test.",
```

Comment:
This gets pretty complex, so it would be nice to see some examples in this doc string.

Basically, I'm looking for a "tar xvzf" or "ps aux" -- things that a newbie can run and have some success with, even if they don't understand all the options available to them.